tests/role_moderation_test.py

```diff
@@ -44,16 +45,16 @@ def test_moderation(fc):
         sirepo.auth_role.ROLE_ADM,
     )
     r = fc.sr_post(
-        'getPendingAndClarifyUserRoleInvites',
+        'getModerationRequestRows',
         PKDict(),
     )
     pkunit.pkeq(len(r.rows), 1)
     fc.sr_post(
-        'approveOrDenyRole',
+        'admModerate',
         PKDict(
             token=r.rows[0].token,
-            approved=True,
+            status='approve',
         ),
     )
-    r = fc.sr_sim_data(sim_type='elegant', sim_name='SPEAR3')
-    pkunit.pkeq(r.models.simulation.name, 'SPEAR3')
+    r = fc.sr_sim_data(sim_type='myapp', sim_name='Scooby Doo')
+    pkunit.pkeq(r.models.simulation.name, 'Scooby Doo')
```

Comment:
You don't need to test Scooby Doo. You can just see if you got back any models:

```py
pkunit.pkok(r.get('models'), 'no models r={}', r)
```

One of the ways to make tests more robust to changes is to avoid testing too much. In this case, if you don't get models back, that's enough. By doing a `get` the test will output the error message instead of getting a KeyError, which is less informative.

Reply:
ad61fc813610afe4d4c57cbe0e4322c503d6ac6c